runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargePlugin.java

```diff
@@ -197,6 +197,8 @@ ItemChargeConfig getConfig(ConfigManager configManager)
 	protected void startUp()
 	{
 		overlayManager.add(overlay);
+		// Update explorer ring charges with value retrieved from varbit value
```

Comment:
What does this comment clarify? This reads exactly the same as the following line, translated to a sentence.